main.js

```diff
@@ -466,7 +466,9 @@ class Student extends User {
     }
     get course () {
         const currentYear = new Date().getFullYear();
-        return `${currentYear - this.admissionYear} курс`;
+        if (this.admissionYear != 2022) {
+            return `${currentYear - this.admissionYear} курс`
+        } else return "1 курс"
```

Comment:
if { return } else { return } is a bad practice - if you go to the IF block with return it's will be really unnecessary to wrap "return" from the outside in else